Lib/test/test_os.py

```diff
@@ -2077,6 +2077,13 @@ def test_file_link(self):
         self.assertTrue(os.path.islink(self.filelink))
         self.check_stat(self.filelink, self.filelink_target)
 
+    @unittest.skipUnless(hasattr(os, 'readlink'), 'needs os.readlink()')
+    def test_readlink_pathlike(self):
+        import pathlib
+        os.symlink(self.filelink_target, self.filelink)
+        filelink = pathlib.Path(self.filelink)
```

Comment:
`FakePath` is used for testing general support of the path protocol, not tied to pathlib.